CHANGELOG.md

```diff
@@ -1,3 +1,83 @@
+# [2022-03-18]
+
+## Release notes
+
+* Deploy Brig before Spar. (#2149)
+* If you are in a federated network of backends (currently beta), you need to update all participating instances at the same time. (#2173)
+
+## API changes
+
+* The `client` JSON object now has an additional field `mls_public_keys`, containing an object mapping signature schemes to public keys, e.g.
+  ```
+  {
+    ...
+    "mls_public_keys": { "ed25519": "GY+t1EQu0Zsm0r/zrm6zz9UpjPcAPyT5i8L1iaY3ypM=" }
+    ...
+  }
+  ```
+  At the moment, `ed25519` is the only supported signature scheme, corresponding to MLS ciphersuite 1.
+
+  When creating a new client with `POST /clients`, the field `mls_public_keys` can be set, and the corresponding public keys are bound to the device identity on the backend, and will be used to verify uploaded key packages with a matching signature scheme.
+
+  When updating a client with `PUT /clients/:client`, the field `mls_public_keys` can also be set, with a similar effect. If a given signature scheme already has a public key set for that device, the request will fail. (#2147)
+
+* Introduce an endpoint for creating an MLS conversation (#2150)
+
+* The `/billing` and `/teams/.*/billing` endpoints are now available on a versioned path (e.g. `/v1/billing`)
+
+   (#2167)
+
+
+## Features
+
+
+* MLS implementation progress:
+
```

Comment:
is this changelog generated?
it's a bit inconsistent sometimes, e.g. extra new lines between bullet points, or bullet point starting with upper/lower case.

Reply:
> is this changelog generated? it's a bit inconsistent sometimes, e.g. extra new lines between bullet points, or bullet point starting with upper/lower case.

yes, [this script](https://github.com/wireapp/wire-server/blob/develop/changelog.d/mk-changelog.sh) digests `/changelog.d` into `CHANGELOG`.